### ISSUE TITLE
Different behaviour for card_declined errors

### ISSUE THREAD

**Issue by erikmolenaarnl** [woothemes/woocommerce-gateway-stripe#62]
Hi!

I noticed with the old **version 2.6.12** when user gets a “card_declined”:

– Order is saved, set to “**Failed**” http://screencast.com/t/IpVGllHzO
– Admin receives a “Failed order” **notification**
– User can try again and again, this will NOT generate new orders (just **1 order instance**)
– After a successful payment the status of the **same order** is set to “Processing”

Now with the new **version 3.0.2** when user gets a “card_declined”:

– Order is saved, set to “**On hold**”
– Admin nor client receives **NO email notifications** of a failed order
– User can try again and again, **every try will generate a new order!**
– After a successful payment another **new order will be generated** which is set to “Processing” http://screencast.com/t/N9YVUIs2KBA6

Could this be a bug? It is much cleaner when only 1 order instance is used, “Failed” is much more appropriate for a declined error, and a failed order notification to admin would be very appropriate as well !

**Comment by quasel** [comment]
Seems to work for me as expected - only on order is created and put on pending payment but i agree it would be a good idea to get a notification about failed payment attempts!

**Comment by erikmolenaarnl** [comment]
I tried again in another WP install, now a 'card_declined' order is indeed set to "**Pending Payment**" (which is appropriate!), user can try again and again and it will remain **ONE** order instance (which is good as well!). I think I messed up with an older release of the plugin, sorry for that.

Only 'unwanted' behavior is there is no more "Failed order" notification e-mail sent out to the admin, which the previous version (2.6.12) did!
